### DISCUSSION TITLE
ARRL DX format for KH6/KL7 is incorrect

### DISCUSSION THREAD

**Discussion by robbomac** [w7sst/MorseRunner#217] [General]
I just discovered MRCE 1.83, and was testing the ARRL DX contest setup with my Hawaii callsign. I entered my exchange as a power level, since Alaska/Hawaii entrants are considered DX, and received an "invalid exchange" error. Can this be easily corrected in a future update?

**Comment by w7sst** [comment]
This discussion has been moved to issue #218
